docs/csharp/language-reference/operators/member-access-operators.md

```diff
@@ -206,6 +209,23 @@ You can omit any of the operands of the `..` operator to obtain an open-ended ra
 
 :::code language="csharp" source="snippets/shared/MemberAccessOperators.cs" id="RangesOptional":::
 
+There are several ways to express range operators. Please consider the descriptions from the following table:
+
+| Range operator expression | Description                                                                          |
+|---------------------------|--------------------------------------------------------------------------------------|
+| `..`                      | All values in the collection.                                                        |
+| `..{end}`                 | Values from the start to the `{end}` exclusively.                                    |
+| `{start}..`               | Values from the `{start}` inclusively to the end.                                    |
+| `{start}..{end}`          | Values from the `{start}` inclusively to the `{end}` exclusively.                    |
+| `^{start}..`              | Values from the `{start}` inclusively to the end starting from the end.              |
```

Comment:
That maybe confusing: my first understanding was that "starting from the end" means "in the reverse order".  Maybe introduce `Length` and use it instead?

Reply:
For me that was hard to grasp because "starting from the end" refers to "`{start}`", while there is text between them. So connection is not immediate.

Reply:
Maybe "counting from the end" rather than "starting from the end"?  What do you both think?

Reply:
I like "counting from the end"